app/models/discussion.rb

```diff
@@ -5,7 +5,16 @@ class Discussion < ActiveRecord::Base
 
   include ReadableUnguessableUrls
   include Translatable
-  include Searchable
+
```

Comment:
No more searchable; the only search vectors we have are discussion search vectors.